lib/dynflow.rb

```diff
@@ -13,6 +13,22 @@
   logger.add level, message, progname, &block
 end
 
+module Dynflow
+end
+
+require 'zeitwerk'
+loader = Zeitwerk::Loader.for_gem
```

Comment:
This looks odd based on the docs: https://github.com/fxn/zeitwerk/tree/main#for_gem, but it somehow works...